src/game.cpp

```diff
@@ -4070,7 +4068,7 @@ void game::debug()
             std::string s;
             s = _( "Location %d:%d in %d:%d, %s\n" );
             s += _( "Current turn: %d; Next spawn %d.\n%s\n" );
-            s += ngettext( "%d monster exists.\n", "%d monsters exist.\n", num_zombies() );
+            s += ngettext( "%d creatures exists.\n", "%d creatures exist.\n", num_creatures() );
```

Comment:
First function parameter probably should be singular - `"%d creature exists.\n"`